### PR TITLE
feat: added colors and typography

### PR DESCRIPTION
**Pull request by SolMendiola**
# :star: Feature

---

#### :pencil2: Description:

Added colors and typography

---

### CODE REVIEW

File: catalog/lib/theme/app_colors.dart [hunk 1]
```diff
@@ -11,37 +10,210 @@ class AppColors extends ColorScheme {
...
+    required this.textColor,
+    required this.info,
+    required this.success,
+    required this.warning,
+    required this.danger,
+  })  : primary = colorScheme.primary as MaterialColor,
+        secondary = colorScheme.secondary as MaterialColor,
+        surface = colorScheme.surface as MaterialColor,
+        onSurface = colorScheme.onSurface as MaterialColor,
+        onPrimary = colorScheme.onPrimary as MaterialColor,
```
Comment (mirland) [inline catalog/lib/theme/app_colors.dart:36]: maybe assert?

Comment (mirland) [inline catalog/lib/theme/app_colors.dart:36]: do you need assign it? I don't think so

File: catalog/gallery/lib/catalog/catalog_scaffold_screen.dart [hunk 2]
```diff
@@ -17,22 +17,22 @@ class CatalogScaffold extends StatelessWidget {
...
           title: Text(
             title,
-            style: TextStyle(color: context.theme.colors.primary),
+            style: TextStyle(color: context.theme.colors.textColor.shade100),
           ),
         ),
-        backgroundColor: context.theme.colors.background,
+        backgroundColor: context.theme.colors.primary.shade200,
         body: SafeArea(
           child: child,
```
Comment (fedecor9) [inline catalog/gallery/lib/catalog/catalog_scaffold_screen.dart:37]: What about adding padding so the items don't look streched?


kind of like this: 
```suggestion
       Padding(
         padding: EdgeInsets.symmetric(horizontal: 16),
         child: child
```

Comment (SolMendiola) [reply catalog/gallery/lib/catalog/catalog_scaffold_screen.dart:37]: I agree, in the next pr i will change it !

**Pull request merged**